lib/gtfsplus/components/GtfsPlusEditor.js

```diff
@@ -135,21 +172,19 @@ export default class GtfsPlusEditor extends Component<Props> {
       visibleRows
     } = this.props
     if (!feedSource) return null
-    const editingIsDisabled = !user.permissions ||
-      !user.permissions.hasFeedPermission(
-        feedSource.organizationId,
-        feedSource.projectId,
-        feedSource.id,
-        'edit-gtfs'
-      )
+    const { editingIsDisabled } = getEditorEnabledState(feedSource, user, feedIsLocked)
     const activeTable = this.GTFS_PLUS_SPEC.find(t => t.id === activeTableId)
     if (!activeTable) {
       console.warn(`Could not locate GTFS+ table with id=${activeTableId}`)
       return null
     }
 
     return (
-      <ManagerPage ref='page'>
+      <ManagerPage itemToLock={editorActions.GTFS_PLUS_EDITOR_LOCK} ref='page'>
+        <ActiveEditorLockManager
```

Comment:
When clicking "re-lock feed" the sidebar disappears! I'm tagging this line since it perhaps is where the issue is coming from, but am not sure what is actually responsible. 

<img width="1552" alt="Screen Shot 2022-09-14 at 9 13 44 AM" src="https://user-images.githubusercontent.com/86619099/190163711-66a32dcd-66c9-4240-8ba4-acd7288baf3e.png">

Reply:
Did you see any message in the console when that happened? 

This is what I see:
![Screen Recording 2022-09-14 at 2 51 32 PM](https://user-images.githubusercontent.com/56846598/190239316-60ca8f95-0d8f-4ab9-af26-5f6f55afdcf4.gif)

Reply:
Apologies, I was on the wrong data tools-server build